firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -856,10 +855,14 @@ class SearchViewController: SiteTableViewController,
         case .openedTabs:
             return filteredOpenedTabs.count
         case .remoteTabs:
-            return model.shouldShowSyncedTabsSuggestions ?
-            filteredRemoteClientTabs.count : 0
+            return if featureFlags.isFeatureEnabled(.firefoxSuggestFeature, checking: .buildAndUser) {
+                model.shouldShowSyncedTabsSuggestions ?
+                filteredRemoteClientTabs.count : 0
```

Comment:
```suggestion
                model.shouldShowSyncedTabsSuggestions ? filteredRemoteClientTabs.count : 0
```
Out of curiosity, how come you're splitting some ternary operators, leaving others on long lines, etc? We should try to have some consistency. Anyway, this one is fine to be one line, lengthwise.